modules/backend/widgets/Filter.php

```diff
@@ -264,6 +265,24 @@ public function onFilterUpdate()
                 $value = post('options.value.' . $scope->scopeName) ?: null;
                 $this->setScopeValue($scope, $value);
                 break;
+
+            case 'clear':
+                foreach ($this->getScopes() as $scope) {
```

Comment:
I think this variable name would have to be different because otherwise it would be overriding the earlier $scope variable wouldn't it?